routing/following_info.hpp

```diff
@@ -83,5 +83,9 @@ class FollowingInfo
   //@{
   turns::PedestrianDirection m_pedestrianTurn;
   //@}
+
+  // Current speed limit. If no info about speed limit then m_speedLimit == 0.
+  std::string m_speedLimit;
```

Comment:
Should it have a default constructor = "0"?

Reply:
Outdated comment. Changed to
`// If no info about speed limit then m_speedLimit == "" and m_speedLimitUnitsSuffix == "".`